### PR TITLE
Added rule for global styles (body, html, etc.)

### PR DESCRIPTION
**Pull request by notoriousb1t**
Added rule() for global rules so typestyle can be used as a drop in replacement for scss or less.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -58,6 +58,16 @@ export function style(...objects: NestedCSSProperties[]) {
 }
 
 /**
+ * Takes CSSProperties and registers it to a global selector (body, html, etc.)
+ */
+export function rule(selector: string, ...objects: NestedCSSProperties[]): void {
```
Comment (basarat) [inline src/index.ts]: Maybe just `CSSProperties` as `&` interpolation and `@media` isn't actually supported by FreeStyle.registerRule.

**Comment by basarat** [comment]
@notoriousb1t I've renamed it to globalRule : https://github.com/typestyle/typestyle/commit/e100cd452fc73ad438bdb6f397f3e069ed4913cc 

Also thought `rawRule` but aborted as its not too raw e.g. has `backgroundColor` instead of `background-color` :rose: I'll cut a release soon :heart:

**Comment by notoriousb1t** [comment]
Very cool.  Probably could have gone with globalStyle as well? I'm glad you moved the test for rule together.  I had them separated mostly for demonstration of the idea.

**Comment by basarat** [comment]
Thanks for your patience. After a few iterations I've gone with `cssRule`. I understand this is 🚲 🏠 (bike shedding) but I can't think of a name that truly conveys its dangers without still showing its advantages. Added docs https://github.com/typestyle/typestyle#css-replacement If you are happy, I'll cut a release?

I'm happy to go back to `rule` as well?

**Comment by notoriousb1t** [comment]
:) I feel like I am overstepping a bit, actually. 

I think cssRule conveys the fact that it is in fact global without being super scary.  The reason I was thinking of this is that it would be good to create a CLI for compiling CSS. This does seem like a departure from your use cases in the README, but I think a lot of folks would be willing to write styles in TypeScript, but not necessarily be able to switch architectures.

**Pull request merged**